### ISSUE TITLE
site.yml references roles that dont exist  (roles/infra-ansible/roles/config-nexus)

### ISSUE THREAD

**Issue by dealyb** [rht-labs/labs-ci-cd#250]

**Comment by dealyb** [comment]
i tried to install this version and the roles directory sesm to have disappeared but it is still being referenced

**Comment by dealyb** [comment]
TASK [set namespace] ***************************************************************************************************************************************************************
ok: [ci-cd-tooling]

TASK [include_role : roles/infra-ansible/roles/config-nexus] ***********************************************************************************************************************
ERROR! the role 'roles/infra-ansible/roles/config-nexus' was not found in /home/ec2-user/labs-ci-cd/labs-ci-cd/roles:/home/ec2-user/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/home/ec2-user/labs-ci-cd/labs-ci-cd

The error appears to have been in '/home/ec2-user/labs-ci-cd/labs-ci-cd/site.yml': line 21, column 15, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    - include_role:
        name: roles/infra-ansible/roles/config-nexus
              ^ here

**Comment by sherl0cks** [comment]
Hmmm - I just cloned a fresh copy, and I didn't hit this issue (but I did hit a different one).

Did you start with `ansible-galaxy install -r requirements.yml --roles-path=roles`? That should the roles directory.

**Comment by dealyb** [comment]
perhaps not, let me try again.

**Comment by sherl0cks** [comment]
OK - I think you will hit an issue once you get past this one though. We just made a major refactor, could be that some stuff is broken.

**Comment by sherl0cks** [comment]
For reference this is the issue I think that you will actually hit once roles are installed.

#251

**Comment by dealyb** [comment]
this is no longer an issue

**Comment by dealyb** [comment]
this is fixed now
